frontend/public/components/filter-toolbar.tsx

```diff
@@ -294,49 +293,51 @@ const FilterToolbar_: React.FC<FilterToolbarProps & RouteComponentProps> = (prop
             )}
           </ToolbarItem>
         )}
-        <ToolbarItem className="co-filter-search--full-width">
-          <ToolbarFilter
-            deleteChipGroup={() => updateLabelFilter([])}
-            chips={[...labelFilters]}
-            deleteChip={(filter, chip: string) =>
-              updateLabelFilter(_.difference(labelFilters, [chip]))
-            }
-            categoryName="Label"
-          >
+        {(!hideToolbar || !hideLabelFilter) && (
```

Comment:
This should be
```suggestion
        {!hideToolbar && (
```
since you could still have a `Name` filter by itself, no?

Reply:
Wondering if `hideToolbar` should return anything at all?  With these changes, we still get empty markup:

```
<div class="pf-c-toolbar" id="filter-toolbar"><div class="pf-c-toolbar__content"><div class="pf-c-toolbar__content-section"></div><div class="pf-c-toolbar__expandable-content" id="filter-toolbar-expandable-content-47"><div class="pf-c-toolbar__group"></div></div></div><div class="pf-c-toolbar__content pf-m-hidden" hidden=""><div class="pf-c-toolbar__group"></div></div></div>
```

Reply:
Oh, I see that's there because of `columnLayout`.  Is there ever `columnLayout` without the filter(s)?

Reply:
@rhamilto it looks like both row filters and col management are both available, as this only seems to apply to the name/label filter as of now.

Reply:
@rhamilto so the markup you are showing is expected...whether or not thats desired is another thing.

Reply:
`hideToolbar` suggests the toolbar will not be present, so outputting empty markup seems kind of silly and is generally something we try to avoid.  If it is deemed necessary, it seems like renaming the prop is in order (maybe `hideFilters`?).

Reply:
@rhamilto there may be empty markup if there are no row filters for the resource. So I think this is the intent as far as I can tell. We have a name/label filter at the top of the search page, right? That is available for all resources in the search page. Each resource, however, may have additional resource specific filters that are not available in the search toolbar, thus those filters are available. I can add code to remove column management...I don't think its important on this page. If you think the name of the variable should change, I can do that so that it reads hideNameLabelFilters since that is what it does.

Reply:
I vote for a prop name change.  `hideToolbar` is misleading.

Reply:
@rhamilto changed name of prop, added a new one for col management, and updated logic